src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
 * `AbstractNormalizer::handleCircularReference` is now final, and receives two optional extra arguments: the format and the context
 * added support for XML comment encoding (encoding `['#comment' => ' foo ']` results `<!-- foo -->`)
+* added optional `int[] $encoderIgnoredNodeTypes` argument to `XmlEncoder::__construct`.
```

Comment:
`... to configure node types to be ignored during encodage`?

Reply:
Yes, better. Done !